postfix/config.sls

```diff
@@ -101,6 +101,27 @@ run-postmap:
       - file: {{ postfix.config_path }}/transport
 {% endif %}
 
+{% if 'tls_policy' in pillar.get('postfix', '') %}
```

Comment:
Hi, you should add this new `tls_policy` to `pillar.example` if it's needed to activate the feature.

Reply:
Done!